chart/apisix/templates/service-gateway.yaml

```diff
@@ -18,6 +18,7 @@ apiVersion: v1
 kind: Service
```

Comment:
Can this service work well?I found that there are some problems:
- it not config `selector`, it will not bind to any pods
- no ports specified, it will not proxy any traffic

Reply:
`selector` is in the end of this file. It is defined in `_helpers.tpl`
`ports` is defined, the value is `gateway` in values.yaml

Reply:
Got it, hmmm, it is really too long